### DISCUSSION TITLE
Index update

### DISCUSSION THREAD

**Discussion by kolumnin** [community/community#118195] [Programming Help]
### Body

I am unable to view the updated index page at https://kolumnin.github.io. My modified file is saved in my repository. I am using Xiaomi Redmi 9A mobile device.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by MarouaneBenbetka** [comment]
Hello Dr @kolumnin  , 

ths issue is your package.json file , it's not in json format  try this file instead , and if it works mark the question as answered please.

`package.json`
```

  {
    "name": "my_package",
    "description": "",
    "version": "1.0.0",
    "main": "index.js",
    "scripts": {
      "test": "echo \"Error: no test specified\" && exit 1"
    },
    "repository": {
      "type": "git",
      "url": "https://github.com/kolumnin/my_package.git"
    },
    "keywords": [],
    "author": "",
    "license": "ISC",
    "bugs": {
      "url": "https://github.com/kolumnin/my_package/issues"
    },
    "homepage": "https://github.com/kolumnin/my_package"
  }

```
